extensions/amp-access/0.1/amp-access.js

```diff
@@ -725,7 +512,10 @@ export class AccessService {
           this.reportViewPromise_ = null;
           throw reason;
         });
+
+    // Why does this happen?
```

Comment:
?

Reply:
Oh, this is existing logic that I did not understand the motivation for, if you had any insight?

Reply:
Primary metering case and pre-rendering. This code specifically registers a view and notifies the pages that might be pre-rendered about this. This might cause some of them to re-render in case the metering is exhausted.

Reply:
cool, added a comment